.github/workflows/typecheck_typeshed_code.yml

```diff
@@ -34,6 +34,8 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: "3.9"
+          cache: 'pip'
+          cache-dependency-path: requirements-tests.txt
```

Comment:
```yaml
          python-version: "3.9"
          cache: 'pip'
          cache-dependency-path: requirements-tests.txt
```
Consider this as a rant about `yaml`: these three lines have three different quotes 😒

Reply:
😅 What's the prefered quote style between `"` and `'` here? Normally I use the [`redhat.vscode-yaml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) plugin which formats to `"` (but I've disabled all my auto-formatters on non-python files in this project).

Reply:
> What's the preferred quote style between `"` and `'` here?

I don't think we have one, and I for one don't care at all, as long as it works :)